src/main/webapp/resources/js/pages/analysis/components/settings/AnalysisDelete.jsx

```diff
@@ -54,7 +54,11 @@ export default function AnalysisDelete() {
           cancelText={i18n("AnalysisDelete.cancel")}
           onConfirm={handleDeleteConfirm}
         >
-          <Button type="danger" className="spaced-top__lg">
+          <Button
+            type="danger"
+            className="spaced-top__lg"
```

Comment:
Let stop using these classes instead just add the inline styled e.g. `marginTop: SPACED_LG`

Reply:
Updated in b7f397facd7f75e5f8d51ba00461f3f813291d2f